### PR TITLE
translation: add European Portugese translation

### PR DESCRIPTION
**Pull request by pfps**
This translation was in the Ubuntu PPA.  It looks as if it never made it into the right place in Solaar.

### CODE REVIEW

File: po/pt.po [hunk 1]
```diff
@@ -0,0 +1,476 @@
...
+msgid "good"
+msgstr "bom"
+
+#: lib/logitech_receiver/i18n.py:38
+msgid "low"
+msgstr "baixo"
+
+#: lib/logitech_receiver/i18n.py:41
+msgid "almost full"
+msgstr "quase cheiro"
```
Comment (FFY00) [inline po/pt.po]: `cheio`

File: po/pt.po [hunk 2]
```diff
@@ -0,0 +1,476 @@
...
+msgid "recharging"
+msgstr "a carregar"
+
+#: lib/logitech_receiver/i18n.py:42
+msgid "invalid battery"
+msgstr "bateria inválida"
+
+#: lib/logitech_receiver/i18n.py:42
+msgid "slow recharge"
+msgstr "carga lenta"
```
Comment (FFY00) [inline po/pt.po]: Maybe `a carregar lentamente`?

File: po/pt.po [hunk 3]
```diff
@@ -0,0 +1,476 @@
...
+msgid "sequence timeout"
+msgstr "tempo limite da sequência"
+
+#: lib/logitech_receiver/i18n.py:45
+msgid "too many devices"
+msgstr "demasiados dispositivos"
+
+#: lib/logitech_receiver/i18n.py:48
+msgid "Bootloader"
+msgstr "Gestor de arranque"
```
Comment (FFY00) [inline po/pt.po]: Maybe we can keep `Bootloader` as it's a widely accepted term, `Gestor de arranque` is not really clear.

File: po/pt.po [hunk 4]
```diff
@@ -0,0 +1,476 @@
...
+msgid "closed"
+msgstr "fechado"
+
+#: lib/logitech_receiver/notifications.py:67
+msgid "open"
+msgstr "aberto"
+
+#: lib/logitech_receiver/notifications.py:67
+msgid "pairing lock is "
+msgstr "tranca de emparelhamento é "
```
Comment (FFY00) [inline po/pt.po]: `o emparelhamente está `

`tranca` is not commonly used in this way in European Portuguese.

File: po/pt.po [hunk 5]
```diff
@@ -0,0 +1,476 @@
...
+
+#: lib/logitech_receiver/notifications.py:192
+msgid "powered on"
+msgstr "com energia ligada"
```
Comment (FFY00) [inline po/pt.po]: `ligado`

File: po/pt.po [hunk 6]
```diff
@@ -0,0 +1,476 @@
...
+msgid "Side Scrolling"
+msgstr "Deslocamento Lateral"
+
+#: lib/logitech_receiver/settings_templates.py:80
+msgid ""
+"When disabled, pushing the wheel sideways sends custom button "
+"events\n"
+"instead of the standard side-scrolling events."
+msgstr ""
+"Quando desactivado, premir a roda para os lados envia eventos de botões "
```
Comment (FFY00) [inline po/pt.po]: `desactivado` is now spelled `desativado`

File: po/pt.po [hunk 7]
```diff
@@ -0,0 +1,476 @@
...
+"personalizados\n"
+"em vez dos eventos de deslocamento lateral standard."
```
Comment (FFY00) [inline po/pt.po]: `standard` > `normais`

File: po/pt.po [hunk 8]
```diff
@@ -0,0 +1,476 @@
...
+
+#: lib/logitech_receiver/settings_templates.py:82
+msgid "Sensitivity (DPI)"
+msgstr "Sensibilidade (DPI)"
+
+#: lib/logitech_receiver/settings_templates.py:83
+msgid "Swap Fx function"
+msgstr "Trocar função Fx"
```
Comment (FFY00) [inline po/pt.po]: `Trocar a função do Fx`

File: po/pt.po [hunk 9]
```diff
@@ -0,0 +1,476 @@
...
+
+#: lib/logitech_receiver/settings_templates.py:84
+msgid ""
+"When set, the F1..F12 keys will activate their special function,\n"
+"and you must hold the FN key to activate their standard function."
+msgstr ""
+"Quando definido, as teclas F1..F12 irão activar as suas funções especiais,\n"
```
Comment (FFY00) [inline po/pt.po]: `activar` is now spelled `ativar`

File: po/pt.po [hunk 10]
```diff
@@ -0,0 +1,476 @@
...
+"e você tem manter a tecla FN carregada para activar as suas funções "
+"standard."
```
Comment (FFY00) [inline po/pt.po]: `standard` > `normais`

File: po/pt.po [hunk 11]
```diff
@@ -0,0 +1,476 @@
...
+msgid "Swap Fx function"
+msgstr "Trocar função Fx"
+
+#: lib/logitech_receiver/settings_templates.py:84
+msgid ""
+"When set, the F1..F12 keys will activate their special function,\n"
+"and you must hold the FN key to activate their standard function."
+msgstr ""
+"Quando definido, as teclas F1..F12 irão activar as suas funções especiais,\n"
+"e você tem manter a tecla FN carregada para activar as suas funções "
```
Comment (FFY00) [inline po/pt.po]: Let's try to avoid `você`.

```
para ativar as funções normais será necessário primir a tecla FN
```

File: po/pt.po [hunk 12]
```diff
@@ -0,0 +1,476 @@
...
+"standard."
+
+#: lib/logitech_receiver/settings_templates.py:87
+msgid ""
+"When unset, the F1..F12 keys will activate their standard function,\n"
+"and you must hold the FN key to activate their special function."
+msgstr ""
+"Quando não definido, as teclas F1..F12 irão activar as suas funções "
```
Comment (FFY00) [inline po/pt.po]: `Quando desativado`.

Also, same comment as above.

File: po/pt.po [hunk 13]
```diff
@@ -0,0 +1,476 @@
...
+"standard,\n"
+"e você tem manter a tecla FN carregada para activar as suas funções "
+"especiais."
+
+#: lib/logitech_receiver/settings_templates.py:89
+msgid "Hand Detection"
+msgstr "Detecção de Mão"
```
Comment (FFY00) [inline po/pt.po]: `Detecção` is now spelled `Deteção`

File: po/pt.po [hunk 14]
```diff
@@ -0,0 +1,476 @@
...
+msgid "Lighting"
+msgstr "Iluminação"
+
+#: lib/logitech_receiver/status.py:161 lib/solaar/ui/window.py:666
+msgid "lux"
+msgstr "lux"
+
+#: lib/solaar/listener.py:95
+msgid "The receiver was unplugged."
+msgstr "O receptor foi desligado."
```
Comment (FFY00) [inline po/pt.po]: `receptor` is now spelled `recetor`

File: po/pt.po [hunk 15]
```diff
@@ -0,0 +1,476 @@
...
+msgid "Permissions error"
+msgstr "Erro de permissões"
+
+#: lib/solaar/ui/__init__.py:70
+#, python-format
+msgid ""
+"Found a Logitech Receiver (%s), but did not have permission to open "
+"it."
+msgstr ""
+"Encontrado um Receptor Logitech (%s), mas não teve permissões para o "
```
Comment (FFY00) [inline po/pt.po]: Same, `recetor`.

File: po/pt.po [hunk 16]
```diff
@@ -0,0 +1,476 @@
...
+"abrir."
+
+#: lib/solaar/ui/__init__.py:72
+msgid ""
+"If you've just installed Solaar, try removing the receiver and "
+"plugging it back in."
+msgstr ""
+"Se você acabou de instalar o Solaar, tente remover o receptor e ligá-lo "
```
Comment (FFY00) [inline po/pt.po]: Same, `recetor`.

File: po/pt.po [hunk 17]
```diff
@@ -0,0 +1,476 @@
...
+"de novo."
+
+#: lib/solaar/ui/__init__.py:74
+msgid "Unpairing failed"
+msgstr "Desemparelhamento falhado"
```
Comment (FFY00) [inline po/pt.po]: Maybe `falhou`?

File: po/pt.po [hunk 18]
```diff
@@ -0,0 +1,476 @@
...
+
+#: lib/solaar/ui/__init__.py:75
+#, python-format
+msgid "Failed to unpair %s from %s."
+msgstr "Falha ao desemparelhar %s de %s."
+
+#: lib/solaar/ui/__init__.py:77
+msgid "The receiver returned an error, with no further details."
+msgstr "O receptor devolveu um erro, sem mais detalhes."
```
Comment (FFY00) [inline po/pt.po]: Same, `recetor`.

File: po/pt.po [hunk 19]
```diff
@@ -0,0 +1,476 @@
...
+
+#: lib/solaar/ui/about.py:39
+msgid ""
+"Shows status of devices connected\n"
+"through wireless Logitech receivers."
+msgstr ""
+"Mostra o estado dos dispositivos ligados\n"
+"através de receptores sem fio da Logitech."
```
Comment (FFY00) [inline po/pt.po]: Same, `recetor`.

File: po/pt.po [hunk 20]
```diff
@@ -0,0 +1,476 @@
...
+#: lib/solaar/ui/pair_window.py:133
+msgid "Pairing failed"
+msgstr "Emparelhamento falhado"
+
+#: lib/solaar/ui/pair_window.py:135
+msgid ""
+"Make sure your device is within range, and has a decent battery "
+"charge."
+msgstr ""
+"Certifique-se que o seu dispositivo está dentro do alcance, e tem "
```
Comment (FFY00) [inline po/pt.po]: No comma.

File: po/pt.po [hunk 21]
```diff
@@ -0,0 +1,476 @@
...
+"uma carga de bateria decente."
```
Comment (FFY00) [inline po/pt.po]: Maybe
```
e tem bateria
```

File: po/pt.po [hunk 22]
```diff
@@ -0,0 +1,476 @@
...
+
+#: lib/solaar/ui/pair_window.py:137
+msgid ""
+"A new device was detected, but it is not compatible with this "
+"receiver."
+msgstr ""
+"Foi detectado um novo dispositivo, mas não é compatível com este "
+"receptor."
```
Comment (FFY00) [inline po/pt.po]: Same, `recetor`.

File: po/pt.po [hunk 23]
```diff
@@ -0,0 +1,476 @@
...
+
+#: lib/solaar/ui/pair_window.py:139
+#, python-format
+msgid "The receiver only supports %d paired device(s)."
+msgstr "O receptor apenas suporta %d dispositivo(s) emparelhados."
```
Comment (FFY00) [inline po/pt.po]: Same, `recetor`.

File: po/pt.po [hunk 24]
```diff
@@ -0,0 +1,476 @@
...
+msgid "Found a new device"
+msgstr "Encontrado um novo dispositivo"
+
+#: lib/solaar/ui/pair_window.py:180
+msgid "The wireless link is not encrypted"
+msgstr "A ligação sem fios não está encriptada"
+
+#: lib/solaar/ui/pair_window.py:197
+msgid "pair new device"
+msgstr "emparelhar o novo dispositivo"
```
Comment (FFY00) [inline po/pt.po]:
```
emparelhar um novo dispositivo
```

File: po/pt.po [hunk 25]
```diff
@@ -0,0 +1,476 @@
...
+msgid ""
+"If the device is already turned on,\n"
+"turn if off and on again."
+msgstr ""
+"Se o dispositivo já está ligado,\n"
+"desligue-o e volte a ligá-lo."
+
+#: lib/solaar/ui/tray.py:55
+msgid "No Logitech receiver found"
+msgstr "Nenhum receptor Logitech encontrado"
```
Comment (FFY00) [inline po/pt.po]: Same, `recetor`.

File: po/pt.po [hunk 26]
```diff
@@ -0,0 +1,476 @@
...
+
+#: lib/solaar/ui/tray.py:62
+msgid "Quit"
+msgstr "Terminar"
+
+#: lib/solaar/ui/tray.py:269
+msgid "no receiver"
+msgstr "nenhum receptor"
```
Comment (FFY00) [inline po/pt.po]: Same, `recetor`.

File: po/pt.po [hunk 27]
```diff
@@ -0,0 +1,476 @@
...
+
+#: lib/solaar/ui/tray.py:288
+msgid "no status"
+msgstr "nenhum estado"
+
+#: lib/solaar/ui/window.py:58
+msgid "The wireless link between this device and its receiver is encrypted."
+msgstr ""
+"A ligação sem fios entre este dispositivo e o seu receptor está encriptada."
```
Comment (FFY00) [inline po/pt.po]: Same, `recetor`.

File: po/pt.po [hunk 28]
```diff
@@ -0,0 +1,476 @@
...
+"\n"
+"For pointing devices (mice, trackballs, trackpads), this is a minor "
+"security issue.\n"
+"\n"
+"It is, however, a major security issue for text-input devices "
+"(keyboards, numpads),\n"
+"because typed text can be sniffed inconspicuously by 3rd parties "
+"within range."
+msgstr ""
+"A ligação sem fios entre este dispositivo e o seu receptor não está "
```
Comment (FFY00) [inline po/pt.po]: Same, `recetor`.

File: po/pt.po [hunk 29]
```diff
@@ -0,0 +1,476 @@
...
+"encriptada.\n"
+"\n"
+"Para dispositivos apontadores (ratos, trackballs, trackpads), este é "
+"um problema de segurança menor.\n"
+"\n"
+"No entanto, é um problema de segurança maior para dispositivos "
+"de entrada de texto (teclados, teclados numéricos).\n"
+"porque o texto teclado pode ser \"cheirado\" de modo imperceptível "
```
Comment (FFY00) [inline po/pt.po]: Maybe
```
pode ser intercetado
```

File: po/pt.po [hunk 30]
```diff
@@ -0,0 +1,476 @@
...
+"por terceiros que estejam dentro do alcance da ligação."
+
+#: lib/solaar/ui/window.py:67 lib/solaar/ui/window.py:71
+msgid "No device paired"
+msgstr "Nenhum dispositivo emparelhado"
+
+#: lib/solaar/ui/window.py:67 lib/solaar/ui/window.py:68
+#, python-format
+msgid "Up to %d devices can be paired to this receiver"
+msgstr "Podem ser emparelhados até %d dispositivos com este receptor"
```
Comment (FFY00) [inline po/pt.po]: Same, `recetor`.

File: po/pt.po [hunk 31]
```diff
@@ -0,0 +1,476 @@
...
+
+#: lib/solaar/ui/window.py:68
+msgid "paired devices"
+msgstr "dispositivos emparelhados"
+
+#: lib/solaar/ui/window.py:72
+msgid "Only one device can be paired to this receiver"
+msgstr "Apenas um dispositivo pode ser emparelhado com este receptor"
```
Comment (FFY00) [inline po/pt.po]: Same, `recetor`.

File: po/pt.po [hunk 32]
```diff
@@ -0,0 +1,476 @@
...
+
+#: lib/solaar/ui/window.py:113
+msgid "Scanning"
+msgstr "A sondar"
```
Comment (FFY00) [inline po/pt.po]:
```
A procurar
```

File: po/pt.po [hunk 33]
```diff
@@ -0,0 +1,476 @@
...
+msgid "Select a device"
+msgstr "Seleccionar um dispositivo"
+
+#: lib/solaar/ui/window.py:511
+msgid "Path"
+msgstr "Caminho"
+
+#: lib/solaar/ui/window.py:513
+msgid "USB id"
+msgstr "id de USB"
```
Comment (FFY00) [inline po/pt.po]:
```
id USB
```

File: po/pt.po [hunk 34]
```diff
@@ -0,0 +1,476 @@
...
+msgid "Wireless PID"
+msgstr "PID de ligação sem fios"
+
+#: lib/solaar/ui/window.py:525
+msgid "Protocol"
+msgstr "Protocolo"
+
+#: lib/solaar/ui/window.py:527
+msgid "Polling rate"
+msgstr "Rácio de votações"
```
Comment (FFY00) [inline po/pt.po]:
```
Frequência de atualização
```

**Comment by pfps** [comment]
Hmm.  I don't speak Portugese at all - this came via @muzena as a translation file that was in the Ubuntu distribution.   If @muzena doesn't speak Portugese either, then I guess this should just be dropped.

**Comment by FFY00** [comment]
I can fix it later.

**Comment by muzena** [comment]
This patch/file came from debian.

**Pull request closed without merging**